index.html

```diff
@@ -6,5 +6,178 @@
     <link href="css/styles.css" rel="stylesheet" />
     <title>Document</title>
```

Comment:
I think your latest changes from Glitch did not get pushed to GitHub, as the code here does not match your code on Glitch, @skwon02. Can you open your project in VS Code and, after you make sure you're on the **poster** branch, do `git push` one more time to make sure this code is updated?

Reply:
Awesome, thank you!